src/pytest_update_test_results/update_test_results.py

```diff
@@ -8,6 +8,18 @@
 def modify_xml(
     original_xml: Path, retest_results: Dict[str, TestReport], new_xml: Path
 ) -> None:
+    """Modifies an XML file containing test results by removing failed test cases that were subsequently
+    passed in a retest.
+
+        Parameters:
+            original_xml (Path): Path to the original XML file containing test results.
+            retest_results (Dict[TestReport]): A dictionary mapping test cases to their retest results.
+            new_xml (Path): The path to the new modified XML file that will be created.
```

Comment:
```suggestion
            original_xml: Path to the original XML file containing test results.
            retest_results: A dictionary mapping test cases to their retest results.
            new_xml: The path to the new modified XML file that will be created.
```

No need to include the types given they are already in the signature.

Btw which docstring format is this?

Reply:
I just checked from the example I used, the docstring format is from [Google](https://google.github.io/styleguide/pyguide.html), but I'll check on confluence if I find the docs format we should be using at ESSS. 
And about the types, the idea was to explain what each parameter does, so I just copied the parameter list from the function definition and the types came along! I'll remove them so it's not so redundant.

Reply:
We use reStructuredText default

Reply:
Ok thanks, will update!